integration_test/tests/connection_tests.rs

```diff
@@ -107,6 +107,7 @@ mod integration {
     // At connection time, when the connection string is processed, if there is a log level specified
     // the logger log level is updated to the connection log level.
     #[test]
+    #[ignore]
```

Comment:
we're skipping this test in evergreen, so I added a skip here as well. I think we should probably open a ticket to look into this, or see if we even want to try to test this going forward.